scout/server/blueprints/institutes/controllers.py

```diff
@@ -171,7 +171,14 @@ def populate_case_obj(case_obj):
         case_obj["assignees"] = [
             store.user(user_email) for user_email in case_obj.get("assignees", [])
         ]
-        case_obj["is_rerun"] = len(case_obj.get("analyses", [])) > 0
+        # Check if case was re-runned
+        analyses = case_obj.get("analyses", [])
+        now = datetime.datetime.now()
+        case_obj["is_rerun"] = (
+            len(analyses) > 1
+            or analyses
```

Comment:
Nice! I worry about the order of checking for existence and use of analyses[0] though; it may be ok but did you test with an old/fake case with empty analyses?

Reply:
One could change the `or analyses` into `or len(analyses) == 1`

Reply:
If you have empty analyses 'analyses' is False and you don't get into the condition

Reply:
Test with an empty "analysis" array: 

![image](https://user-images.githubusercontent.com/28093618/114844408-10c9ee80-9ddb-11eb-92c7-8eef3a13daa6.png)

Does not crash and of course does not show 'rerun' badge: 🆗 

![image](https://user-images.githubusercontent.com/28093618/114844504-23dcbe80-9ddb-11eb-8d0d-a8c54109de01.png)